resources/definitions/renkforce_rf100.def.json

```diff
@@ -0,0 +1,231 @@
+{
+    "id": "RF100",
+    "version": 2,
+    "name": "renkforce RF100",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "author": "Simon Peter (based on RF100.ini by Conrad Electronic SE)",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "manufacturer": "renkforce",
+        "platform_offset": [0, 0, 0],
```

Comment:
There is no platform mesh for this printer. This line doesn't do anything at the moment. Perhaps you'd like to add a platform mesh? Or otherwise we should remove this.